tests/codegen/overaligned-constant.rs

```diff
@@ -0,0 +1,31 @@
+// GVN may create indirect constants with higher alignment than their type requires. Verify that we
+// do not ICE during codegen, and that the LLVM constant has the higher alignment.
+//
+// compile-flags: -O -Zmir-enable-passes=+GVN -Cdebuginfo=2
```

Comment:
I'm wary of this combination of flags. Why do we need `-Cdebuginfo=2`? Can we use `-Zmir-opt-level=0` so this doesn't depend on which MIR opts `-O` enables?

Reply:
Nice. Thanks.